PluginDirectories/1/fpm.bundle/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "categories" : [
+    "Utilities",
+    "Package Management",
```

Comment:
Not sure I want to add two whole new top-level categories, but this doesn't really fit in the existing categories.

Would it be weird to have a "meta" category for flashlight-related stuff (e.g. the Flashlight plugin search plugin)?